Ã— atuned.service - A-Tune Daemon
     Loaded: loaded (/usr/lib/systemd/system/atuned.service; disabled; vendor preset: disabled)
     Active: failed (Result: exit-code) since Thu 2023-06-08 13:01:01 UTC; 392ms ago
    Process: 4303 ExecStart=/usr/bin/atuned (code=exited, status=255/EXCEPTION)
   Main PID: 4303 (code=exited, status=255/EXCEPTION)

Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=info msg="valid network : ens6" file="config.go:321"
Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=info msg="Connecting to DB: /var/lib/atuned/atuned.db" file="sqlstore.go:47"
Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=info msg="initializing service: monitor" file="atuned.go:126"
Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=info msg="initializing service: pyengine" file="atuned.go:126"
Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=info msg="load profile service successfully\n" file="profile.go:111"
Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=info msg="Create opt.profile service successfully\n" file="atuned.go:65"
Jun 08 13:01:01 localhost.localdomain atuned[4303]: time="2023-06-08T13:01:01Z" level=error msg="cmd.Run() analysis failed with: exit status 1" file="pyservice.go:76"
Jun 08 13:01:01 localhost.localdomain systemd[1]: atuned.service: Main process exited, code=exited, status=255/EXCEPTION
Jun 08 13:01:01 localhost.localdomain systemd[1]: atuned.service: Failed with result 'exit-code'.
Jun 08 13:01:01 localhost.localdomain systemd[1]: Failed to start A-Tune Daemon.
